workers/unity/Packages/com.improbable.gdk.transformsynchronization/Components/BufferedTransform.cs

```diff
@@ -7,7 +7,7 @@ public struct BufferedTransform : IBufferElementData
     {
         public Vector3 Position;
         public Vector3 Velocity;
-        public Quaternion Orientation;
+        public UnityEngine.Quaternion Orientation;
```

Comment:
yes, to avoid conflicts with the `Quaternion` defined in the transform module schema - which now gets generated in the `Improbable.Gdk.TransformSynchronization` namespace